.github/workflows/tests.yml

```diff
@@ -0,0 +1,155 @@
+name: tests
+on:
+  push:
+    branches-ignore: [2.0]
+  workflow_dispatch:
+
+jobs:
+  # test-browser:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - uses: browser-actions/setup-chrome@v1
+  #       with:
+  #         chrome-version: stable
+  #       id: setup-chrome
+  #     - run: |
+  #         echo Installed Chrome version: ${{ steps.setup-chrome.outputs.chrome-version }}
+  #         ${{ steps.setup-chrome.outputs.chrome-path }} --version
+  #
+  #         CHROME_VERSION=$(echo "${{ steps.setup-chrome.outputs.chrome-version }}" cut -f 1 -d ' ' | cut -d '.' -f 1)
+  #
+  #         CHROMEDRIVER_VERSION=$(curl --location --fail --retry 10 http://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROME_VERSION)
+  #
+  #         echo "chromedriver_version=$CHROMEDRIVER_VERSION" >> "$GITHUB_ENV"
+  #     - uses: nanasess/setup-chromedriver@v2
```

Comment:
pin 3rdParty actions to a SHA in case someone evil gets control of the repo

Reply:
done